File: TheaterSuggestions/Java/SeatsSuggestionsTests/src/test/java/com/baasie/SeatsSuggestionsTests/NotSuggestionMatchedExpectation.java
package com.baasie.SeatsSuggestionsTests;

import java.util.ArrayList;

public class NotSuggestionMatchedExpectation extends SuggestionMade {
    public NotSuggestionMatchedExpectation(int partyRequested, PricingCategory pricingCategory) {
        super(new ArrayList<>(), partyRequested, pricingCategory);
    }
}

File: TheaterSuggestions/CSharp/SeatsSuggestions/NotSuggestionMatchedExpectation.cs
using System.Collections.Generic;

namespace SeatsSuggestions
{
    internal class NotSuggestionMatchedExpectation : SuggestionMade
    {
        public NotSuggestionMatchedExpectation(int partyRequested, PricingCategory pricingCategory) : base(
            partyRequested, pricingCategory, new List<Seat>())
        {
        }
    }
}